File: java/parser/src/main/java/gov/lbl/parser/domain/Function_argument.java
package gov.lbl.parser.domain;

import java.util.Collection;

public class Function_argument {
    private Name function_name;
    private Named_arguments named_arguments;
    private Expression expression;

    public Function_argument(Name function_name, Named_arguments named_arguments, Expression expression) {
        this.function_name = function_name;
        this.named_arguments = named_arguments;
        this.expression = expression;
    }
}

File: jsParser/domain/Function_argument.js
class Function_argument {
  constructor (function_name, named_arguments, expression) {
    function_name != null ? this.function_name = function_name : ''
    named_arguments != null ? this.named_arguments = named_arguments : ''
    expression != null ? this.expression = expression : ''
  }
}
exports.Function_argument = Function_argument
